package-system/squish-ccr/CMakeLists.txt

```diff
@@ -107,11 +109,18 @@ set(SQUISH_PUBLIC_INCLUDE_FILES
 )
 
 add_library(squish-ccr SHARED ${SQUISH_SOURCE_FILES} ${SQUISH_HEADER_FILES} ${SQUISH_INLINE_FILES})
-target_include_directories(squish-ccr PRIVATE ${CMAKE_CURRENT_LIST_DIR})
-target_compile_definitions(squish-ccr PRIVATE SQUISH_USE_SSE=2 SQUISH_USE_CPP SQUISH_USE_CCR)
+
 
 if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-    target_compile_options(squish-ccr PRIVATE -msse2 -Wno-unused-value)
+    if(CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64")
+        target_include_directories(squish-ccr PRIVATE ${CMAKE_CURRENT_LIST_DIR} /data/workspace/sse2neon)
+        target_compile_definitions(squish-ccr PRIVATE SQUISH_USE_NEON=1 SQUISH_USE_SSE=2 SQUISH_USE_CPP SQUISH_USE_CCR)
+        target_compile_options(squish-ccr PRIVATE -march=armv8-a+fp+simd+crypto+crc -Wno-unused-value -Wno-c++20-extensions -Wno-shift-op-parentheses -Wno-tautological-constant-out-of-range-compare -Wno-array-bounds -Wno-switch)
```

Comment:
What is with this architecture?
`a+fp+simd+crypto+crc`?

What is `a`?

Reply:
Its actually 'armv8-a' for the cpu, followed by the options. That line is from the guide for sse2neon which is the project that is used to translate the sse calls to neon:
https://github.com/DLTcollab/sse2neon/blob/master/README.md

Reply:
Thanks for the info!
I was wondering how it combined together.